### ISSUE TITLE
No Error when using too Small Embedding Layer. 

### ISSUE THREAD

**Issue by lysecret2** [keras-team/keras#10694]
Hey,
I am using tf Keras with tensorflow 1.9 and tf data input. I have a seuqence model and I am using an embedding layer. Now i observed the following problem: 
When you use a "too small" embedding layer. So say with 100 Embeddings and you have more tokens than that. It doesnt throw an error or an exception but the loss goes to nan. 

Best.
